sa/sa.go

```diff
@@ -97,11 +105,18 @@ func NewSQLStorageAuthority(
 ) (*SQLStorageAuthority, error) {
 	SetSQLDebug(dbMap, logger)
 
+	failedAddCertRLTransactions := prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "failedAddCertRLTransactions",
```

Comment:
prometheus_idiom_is_snakes

Reply:
(this is an idiom we are pretty bad at following, we have a good mix throughout the codebase of snake and camel)

Reply:
It's true, although a lot of the confusion is holdover from the old statd style of collection, where we used camels extensively. Our general approach has been to convert names over as we switch code to the native Prometheus API, but there are a lot of metrics we've just never touched since then.

It would be nice to do a once-over and convert all the remaining uses of the statsd-style APIs to Prometheus-style, and get rid of the conversion code altogether.

Reply:
> It would be nice to do a once-over and convert all the remaining uses of the statsd-style APIs to Prometheus-style, and get rid of the conversion code altogether.

@jsha Will you file an issue for that?

I can't ever remember which style to use and when I check the codebase the mix makes it hard to intuit the right choice.

Reply:
Changed to snake-case in 9cff4a6

Reply:
:+1: https://github.com/letsencrypt/boulder/issues/4591